cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -0,0 +1,165 @@
+package workerqueue
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/astronomer/astro-cli/astro-client"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+	"github.com/pkg/errors"
+)
+
+var (
+	ErrWorkerQueueDefaultOptions = errors.New("failed to get worker-queue default options")
+	ErrInvalidWorkerQueueOption  = errors.New("worker-queue option is invalid")
+	ErrCannotUpdateExistingQueue = errors.New("worker-queue exists")
+)
+
+func Create(ws, deploymentID, name string, isDefaultWQueue bool, wQueueMin, wQueueMax, wQueueConcurrency int, client astro.Client, out io.Writer) error {
+	var (
+		requestedDeployment       astro.Deployment
+		err                       error
+		errHelp                   string
+		workerQueueToCreate       *astro.WorkerQueue
+		wQueueListToCreate        []astro.WorkerQueue
+		workerQueueDefaultOptions astro.WorkerQueueDefaultOptions
+		pool                      astro.NodePool
+		nodePoolIDForWorkerQueue  string
```

Comment:
```suggestion
		nodePoolID  string
```
nitpick: do we have to append WorkerQueue to the variable names, given that package itself is called worker queue, isn't it implicit that Create method in this package will have variables related to the worker queue, same for other variables naming as well

Reply:
Agreed. I will clean up the variable names to not have WorkerQueue in them.